### PR TITLE
Grid + GridCell

### PR DESCRIPTION
**Pull request by stevenyuen**
**Description**
- `GridCell`s on same row are automatically made to be the same height
- There is no more extra spacing on the bottom row of the `Grid`
- Added basic appearance-related props to `GridCell` including: `backgroundColor`, `backgroundImage`, `border` and `padding`
- Updated Docs for `Grid` and `GridCell`

**Tests**
- Updated Visual tests

### CODE REVIEW

File: src/GridCell/index.js [hunk 1]
```diff
@@ -40,11 +71,27 @@ import withDynamicResponsiveProps from '../utils/withDynamicResponsiveProps'
 
 export default class GridCell extends React.Component {
    static propTypes = {
+      backgroundColor: PropTypes.string,
```
Comment (asabhaney) [inline src/GridCell/index.js]: What about: `backgroundImage`, `backgroundPosition `, `backgroundSize `?

File: src/GridCell/specAppearance.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+   description: 'css equivalent value',
+   responsive: true
+}, {
+   name: 'borderRight',
+   type: 'String',
+   default: '-',
+   description: 'css equivalent value',
+   responsive: true
+}, {
+   name: 'borderTop',
```
Comment (asabhaney) [inline src/GridCell/specAppearance.js]: `borderRadius`?

File: src/GridCell/index.js [hunk 3]
```diff
@@ -40,11 +71,27 @@ import withDynamicResponsiveProps from '../utils/withDynamicResponsiveProps'
...
+      borderRight: OIOResponsiveObjectPropType,
+      borderBottom: OIOResponsiveObjectPropType,
+      borderLeft: OIOResponsiveObjectPropType,
+      boxShadow: OIOResponsiveObjectPropType,
       children: PropTypes.node,
       className: PropTypes.string,
       colspan: OIOResponsiveObjectPropType,
       id: PropTypes.string,
-      style: PropTypes.object
+      padding: OIOResponsiveObjectPropType,
```
Comment (asabhaney) [inline src/GridCell/index.js]: None of the padding seem to be in the specs/docs?

File: src/GridCell/specAppearance.js [hunk 4]
```diff
@@ -0,0 +1,67 @@
+export default [{
+   name: 'backgroundColor',
+   type: 'String',
+   default: '-',
+   description: 'css equivalent value',
+   responsive: true
+}, {
+   name: 'backgroundImage',
```
Comment (asabhaney) [inline src/GridCell/specAppearance.js]: Perhaps a specimen that displays the background image related props would be helpful?

Comment (stevenyuen) [reply src/GridCell/specAppearance.js]: Im going to do this in another PR if that is ok. I filed an issue. #28

**Pull request merged**